modules/apps/portal-search/portal-search-web/src/main/java/com/liferay/portal/search/web/internal/result/display/context/builder/SearchResultSummaryDisplayContextBuilder.java

```diff
@@ -932,9 +932,14 @@ private void _checkViewURL(
 			ClassName className = _classNameLocalService.getClassName(
 				classNameId);
 
-			_buildViewURL(
-				className.getClassName(), classPK,
-				searchResultSummaryDisplayContext);
+			AssetRendererFactory<?> assetRendererFactory =
+				getAssetRendererFactoryByClassName(className.getClassName());
+
+			if (assetRendererFactory != null) {
```

Comment:
Will sourceFormatter allow us to just do `if (getAssetRendererFactoryByClassName(className.getClassName()) != null)`?

Reply:
It does! Adding that in 👍